recipe/recipe.yaml

```diff
@@ -1,58 +1,101 @@
-  name: ${{ name|lower }}
+recipe:
+  name: sparrow-packages
   version: ${{ version }}
 
 source:
   url: https://github.com/man-group/sparrow/archive/${{ version }}.tar.gz
-  sha256: 118feef83085efe362001d9f545c88d8a1518b9036b8641ca1142c182f58dc6f
+  sha256: b50c88be9bc0f3e408299ef58a8a7243b54d31cc9a5610234266f74d0c237920
 
 build:
   number: ${{ build_number }}
 
-requirements:
-  run_exports:
-    - ${{ pin_subpackage('sparrow', upper_bound='x.x') }}
-  build:
-    - ${{ compiler('cxx') }}
-    - ${{ stdlib('c') }}
-    - cmake
-    - if: unix
-      then: make
-  host:
-    - howardhinnant_date >=3,<4
-  #run:
-    ## howardhinnant_date has run_exports
-
-tests:
-  - script:
-    - if: unix
-      then:
-        - test -f ${PREFIX}/include/sparrow/buffer/buffer.hpp
-        - test -f ${PREFIX}/include/sparrow/config/sparrow_version.hpp
-        - test -f ${PREFIX}/share/cmake/sparrow/sparrowConfig.cmake
-        - test -f ${PREFIX}/share/cmake/sparrow/sparrowConfigVersion.cmake
-        - test -f ${PREFIX}/share/cmake/sparrow/sparrowTargets.cmake
-    - if: win
-      then:
-        - if not exist %LIBRARY_PREFIX%\include\sparrow\buffer\buffer.hpp (exit 1)
-        - if not exist %LIBRARY_PREFIX%\include\sparrow\config\sparrow_version.hpp (exit 1)
-        - if not exist %LIBRARY_PREFIX%\share\cmake\sparrow\sparrowConfig.cmake (exit 1)
-        - if not exist %LIBRARY_PREFIX%\share\cmake\sparrow\sparrowConfigVersion.cmake (exit 1)
-        - if not exist %LIBRARY_PREFIX%\share\cmake\sparrow\sparrowTargets.cmake (exit 1)
+outputs:
+  - package:
+      name: sparrow
+      version: ${{ version }}
+    build:
+      script: build
+    requirements:
+      run_exports:
+        - ${{ pin_subpackage('sparrow', upper_bound='x.x') }}
+      build:
+        - ${{ compiler('cxx') }}
+        - ${{ stdlib('c') }}
+        - cmake
+        - if: unix
+          then: make
+      host:
+        - howardhinnant_date >=3,<4
+      #run:
+        # howardhinnant_date has run_exports
+    tests:
+      - script:
+        - if: unix
+          then:
+            - test -f ${PREFIX}/include/sparrow/buffer/buffer.hpp
+            - test -f ${PREFIX}/include/sparrow/config/sparrow_version.hpp
+            - test -f ${PREFIX}/share/cmake/sparrow/sparrowConfig.cmake
+            - test -f ${PREFIX}/share/cmake/sparrow/sparrowConfigVersion.cmake
+            - test -f ${PREFIX}/share/cmake/sparrow/sparrowTargets.cmake
+        - if: win
+          then:
+            - if not exist %LIBRARY_PREFIX%\include\sparrow\buffer\buffer.hpp (exit 1)
+            - if not exist %LIBRARY_PREFIX%\include\sparrow\config\sparrow_version.hpp (exit 1)
+            - if not exist %LIBRARY_PREFIX%\share\cmake\sparrow\sparrowConfig.cmake (exit 1)
+            - if not exist %LIBRARY_PREFIX%\share\cmake\sparrow\sparrowConfigVersion.cmake (exit 1)
+            - if not exist %LIBRARY_PREFIX%\share\cmake\sparrow\sparrowTargets.cmake (exit 1)
+  - package:
+      name: sparrow-dev
```

Comment:
Could we name this `sparrow-devel`? This is the suffix we've been settling on in recent years for such packages.